misc/xml-to-record-converter/src/main/java/io/ballerina/xmltorecordconverter/Constants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
```suggestion
 *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org)
 *
 *  WSO2 LLC. licenses this file to you under the Apache License,
```

Reply:
Apply the same to all the relevant files

Reply:
Seems It's LCC now not LLC

Reply:
Also, the url has to be ".com" not ".org"